### ISSUE TITLE
Image quality degradation over past weeks

### ISSUE THREAD

**Issue by Lmaookay04** [lllyasviel/stable-diffusion-webui-forge#2280]
I run forge webui from a custom notebook in Google colab, I use the A100 GPU (NVIDIA), it has 40GB VRAM. Using for continuously since the support of flux has made it my preference.

 Over the past few weeks though, (with no changes to my routine) attempting similar tasks now nearly always results in terrible image quality that was not present in the earlier days (like early August to early September) of Flux support. The image issues usually are along the lines of blurry, artifact ridden, pixelated, unfinished, uncanny… and other similarities of that nature. 

Interestingly though, reverting to a previous commit does not solve the issue… Making me think maybe it’s a dependency within the colab environment, or some update to their memory, GPU/AI architecture, or other changes variable related to processing on forge, maybe? I don’t know. 

I’m seeking any recommendations or ideas that anyone has about this issue, particularly if you have similar troubles and have potentially found some workaround? Thank you all in advance.

++ If you have a commit recommendation please share as I will be happy to try!

**Comment by derfasthirnlosenick** [comment]
Pls provide setting/prompt/checkpoint for a test case that looks different on your system with two different versions.

**Comment by Eroy09** [comment]
> Pls provide setting/prompt/checkpoint for a test case that looks different on your system with two different versions.

Will do, give me a few hours and i'll launch it up

**Comment by Eroy09** [comment]
- Below are screenshots containing my entire colab notebook workflow for forge webui and the codes' outputs. - 
![Screenshot 2024-11-09 091325](https://github.com/user-attachments/assets/ee98cefd-e5b7-4743-9bf2-8ba34c0a097b)
![Screenshot 2024-11-09 091415](https://github.com/user-attachments/assets/4555eaa2-d5c2-4d67-a226-4fed112f69fa)
![Screenshot 2024-11-09 091449](https://github.com/user-attachments/assets/76106683-7d94-42a1-82ea-34c233f225ec)
![Screenshot 2024-11-09 091531](https://github.com/user-attachments/assets/fd1b83f2-83c2-410e-ac47-b4f41185ae4f)
![Screenshot 2024-11-09 091547](https://github.com/user-attachments/assets/b16cdcd3-2258-4bcf-99cd-01bdaff0b40a)
![Screenshot 2024-11-09 091606](https://github.com/user-attachments/assets/9980ea18-a750-4346-a59e-86961c50731e)
![Screenshot 2024-11-09 092735](https://github.com/user-attachments/assets/0dd1acda-6020-4a23-bc42-209976f4861e)
![Screenshot 2024-11-09 092752](https://github.com/user-attachments/assets/7414be4e-7b26-4a01-92ab-186fc943c035)
![Screenshot 2024-11-09 093545](https://github.com/user-attachments/assets/052506ef-eb78-4086-8bd6-8dc562e99d19)

- Below is a .txt file containing both the latest versions' and a mid august commit's output logs from launch. -
[Logs.txt](https://github.com/user-attachments/files/17688556/Logs.txt)

- Below is a Screen shot from the UI page, used for both generations. -
![Screenshot 2024-11-09 092735](https://github.com/user-attachments/assets/dd885c06-fdcd-4174-aff4-6ed615b20c23)

- Below are both the 1st and 2nd versions' generation outputs, i used the (PNG Info / Send to txt2Img) From the first generation to generate the second output. I also set "GPU weights" to the same value for both. -

(1st) - ![00000-4132367344](https://github.com/user-attachments/assets/d159f96a-5ab2-4ce1-9045-80c761d6e399)

(2nd) - ![00001-4132367344 (2ndVersion)png](https://github.com/user-attachments/assets/c9aa3c13-702e-4782-8826-b79322acae7b)

**Comment by derfasthirnlosenick** [comment]
I didn't do a binary comparison, but the pics look the same... am i missing something?

**Comment by Eroy09** [comment]
> I didn't do a binary comparison, but the pics look the same... am i missing something?

Yeah they're the mostly similar, i'm more referring to their quality, it's really bad, and this is really uncommon for flux, i just did two generations to show the issues is persistent across commits via google colab. 

P.S. - I went to do a comparison in comfyui (sure that it was going to be way better, because just yesterday i was doing reproduction comparisons, and comfyui was producing great images from the exact parameters that were given to forge, forge though handled those generations poorly.) but starting today, comfyui is now producing for me the same quality issues, and was not before. It's like the forge situation all over again, but forge started acting funny for me around Oct 20. I also reproduced similar issues in huggingface spaces

**Comment by derfasthirnlosenick** [comment]
> > I didn't do a binary comparison, but the pics look the same... am i missing something?
> 
> Yeah they're the mostly similar, i'm more referring to their quality, it's really bad, and this is really uncommon for flux, i just did two generations to show the issues is persistent across commits via google colab.
> 
> P.S. - I went to do a comparison in comfyui (sure that it was going to be way better, because just yesterday i was doing reproduction comparisons, and comfyui was producing great images from the exact parameters that were given to forge, forge though handled those generations poorly.) but starting today, comfyui is now producing for me the same quality issues, and was not before. It's like the forge situation all over again, but forge started acting funny for me around Oct 20. I also reproduced similar issues in huggingface spaces

You can just fall back to an old commit and see if the same prompt/settings generates something different.

**Comment by Eroy09** [comment]
> > > I didn't do a binary comparison, but the pics look the same... am i missing something?
> > 
> > 
> > Yeah they're the mostly similar, i'm more referring to their quality, it's really bad, and this is really uncommon for flux, i just did two generations to show the issues is persistent across commits via google colab.
> > P.S. - I went to do a comparison in comfyui (sure that it was going to be way better, because just yesterday i was doing reproduction comparisons, and comfyui was producing great images from the exact parameters that were given to forge, forge though handled those generations poorly.) but starting today, comfyui is now producing for me the same quality issues, and was not before. It's like the forge situation all over again, but forge started acting funny for me around Oct 20. I also reproduced similar issues in huggingface spaces
> 
> You can just fall back to an old commit and see if the same prompt/settings generates something different.

Thats what the 2 images are, the latest version at the time and and an old commit. When i take old generations that were good and feed the parameters back now no matter what it commit it seems it just wont reproduce the image
